### PR TITLE
Feature/17626 add ticket selection

### PR DESCRIPTION
**Pull request by julesian**
Adds multiple ticket selection UI

### CODE REVIEW

File: Sources/TripKit/categories/NSNumber+Formatter.swift [hunk 1]
```diff
@@ -19,6 +19,20 @@ extension NSNumber {
     return formatter
   }()
   
+  // TODO: Better name / distinction from toMoneyString
```
Comment (nighthawk) [inline Sources/TripKit/categories/NSNumber+Formatter.swift:22]: Can you add a comment why we have this as an alternative to `toMoneyString`? Would it maybe make more sense to add a parameter to that existing method to toggle the different behaviours? Seems like the only differences are the fraction digits.

File: Sources/TripKitUI/views/cell accessory/ButtonAccessoryView.swift [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+//  Created by Jules Gilos on 7/27/22.
+//  Copyright © 2022 SkedGo Pty Ltd. All rights reserved.
+//
+
+import TripKit
+
+import RxSwift
+import RxCocoa
+
+class ButtonAccessoryView: UIView {
```
Comment (nighthawk) [inline Sources/TripKitUI/views/cell accessory/ButtonAccessoryView.swift:14]: As discussed separately, this isn't needed in this repo, and can be just in the TripGo repo.

File: Sources/TripKitUI/views/cell accessory/QuantityStepperAccessoryView.swift [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+//
+//  Created by Jules Gilos on 7/27/22.
+//  Copyright © 2022 SkedGo Pty Ltd. All rights reserved.
+//
+
+import UIKit
+
+import RxSwift
+
+class QuantityStepperAccessoryView: UIView {
```
Comment (nighthawk) [inline Sources/TripKitUI/views/cell accessory/QuantityStepperAccessoryView.swift:13]: As discussed separately, this isn't needed in this repo, and can be just in the TripGo repo.

**Review by nighthawk — CHANGES_REQUESTED** [review]

File: Sources/TripKit/categories/NSNumber+Formatter.swift (hunk 1 above)
Comment (julesian) [reply Sources/TripKit/categories/NSNumber+Formatter.swift:22]: was initially thinking of adding parameter to include decimals and let 0 decimal places be the default. But I thought of making a new one for the purpose of having new method caller when it comes to displaying of detailed prices / currency in single unadjustable format (2 decimal places and currency sign) to give devs context of the format used.

Can have the parameter sending within the extension and have it called from functions with context.

**Review by nighthawk — APPROVED** [review]

**Pull request merged**